cityofaustin/atd-moped
# Remove unused status id fields

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/9747

This PR cleans up the deprecated `status_id` columns left behind in the soft delete epic. The `status_id` has also been removed from the tables metadata (and event triggers for notes, components, partners, types, personnel, phases, and milestones).

I'd also like some dev feedback on the down migrations - the multi-statement Hasura-generated ones are a bit goofy since we originally added `status_id` columns like this:
```sql
ALTER TABLE "public"."moped_proj_personnel" ADD COLUMN "status_id" integer NOT NULL DEFAULT 0;
```
I'll happily update them if we have strong opinions about it.

## Testing
**URL to test:**
https://md-9747-remove-unused-status-id-fields--atd-moped-main.netlify.app/moped/

**Steps to test:**
Add/remove each of the following types of records to make sure they are still soft-deletable:
- Project notes (Project Comments tab)
- Project Components, Project Subcomponents and Project Feature (Project Map tab -> Add/remove Type - Bike Lane, Subtype - Buffered, Subcomponents - Raised pavement markers)
- Project Partners (Project Summary page)
- Project Types (Project Summary page)
- Project Personnel (Project Team tab)
- Project Phases (Project Timeline tab)
- Project Milestones (Project Timeline tab)
- Moped User (Staff page)

---
#### Ship list
